classes/class-wc-rest-connect-account-settings-controller.php

```diff
@@ -45,7 +45,8 @@ public function get() {
 				'master_user_login' => is_a( $master_user, 'WP_User' ) ? $master_user->user_login : '',
 				'master_user_email' => $email,
 				'payment_methods' => $this->payment_methods_store->get_payment_methods(),
-			)
+				'warnings' => ! $payment_methods_fetched ? array( 'payment_methods' => 'Encountered an error while retrieving stored payment methods.' ) : array(),
```

Comment:
The WP way is to test the positive case, not the negative. ([#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#ternary-operator))

Please wrap the error message string for translation.

How about "There was a problem fetching your saved credit cards." instead?